Library/Homebrew/formula.rb

```diff
@@ -1288,9 +1290,18 @@ def system(cmd, *args)
           end
           wr.close
 
-          while buf = rd.gets
-            log.puts buf
-            puts buf
+          if verbose
+            while buf = rd.gets
+              log.puts buf
+              puts buf
+            end
+          else # travis
+            while buf = rd.gets
+              log.puts buf
+              print "."
```

Comment:
It's unclear to me when this will print; after every new write to the output? That seems like it's going to print way too much; it should probably have a timer and print e.g. no more than once a minute.